### ISSUE TITLE
Clicking on links from failed test's stacktrace opens files with wrong location on Linux

### ISSUE THREAD

**Issue by cpetrov** [boogie666/atom-mocha#5]
My project is located in ~/git/project. Failed tests stack trace shows paths which would be relative to the home directory:

```
(git/project/test/...spec.js:44:1)
```

And when clicking on an item, the file is opened, but relative to the project and not the home directory:

```
~/git/project/git/project/test/...spec.js
```

From your gif I see those paths are resolved absolutely on Windows: `at C:\cygwin64\...`

Used in this view: https://github.com/boogie666/atom-mocha/blob/master/src/containers/ErrorDisplay.js#L34. See `frame.source` and `frame.fileName`.

**Comment by poelstra** [comment]
Having the same problem (running Ubuntu). It seems the path returned by error-stack-frame is not an absolute path, but is relative to my home dir (e.g. `/home/username`).

Haven't tried running it on a file outside my home dir, though.

**Comment by erelsgl** [comment]
This happened to me too. I am also on Ubuntu.

https://github.com/boogie666/atom-mocha/issues/12

**Comment by boogie666** [comment]
This should be fixed as of version 0.0.9.

I'm gonna close this issue, but feel free to reopen if this is still a problem.

Cheers,
Boogie
